instrumentation/grpc-1.5/src/main/java/io/opentelemetry/auto/instrumentation/grpc/client/GrpcClientDecorator.java

```diff
@@ -28,12 +29,14 @@
       OpenTelemetry.getTracerProvider().get("io.opentelemetry.auto.grpc-1.5");
 
   public Span onClose(final Span span, final io.grpc.Status status) {
+    onComplete(span, GrpcHelper.statusFromGrpcStatus(status), status.getCause());
+    return span;
+  }
 
-    span.setAttribute("status.code", status.getCode().name());
-    span.setAttribute("status.description", status.getDescription());
-
-    onError(span, status.getCause());
-    span.setStatus(GrpcHelper.statusFromGrpcStatus(status));
+  @Override
+  public Span onError(Span span, Throwable throwable) {
+    assert span != null;
+    onClose(span, Status.fromThrowable(throwable));
```

Comment:
i think calling straight to `onComplete` here would be a bit clearer
```suggestion
    onComplete(span, Status.fromThrowable(throwable), throwable);
```

Reply:
Yeah not sure if I have a preference. Happy to make the change, but want to confirm you think it's better even though it needs to be this longer version (convert to otel status)?

`onComplete(span, GrpcHelper.statusFromGrpcStatus(Status.fromThrowable(throwable)), throwable);`

Reply:
oh yeah, that's not great, keep it as is 👍